### PR TITLE
Pin reset button will now be diaplayed from the very first attempt

### PR DESCRIPTION
**Pull request by chandanvarma009**
Pin reset button is now visible all the time

### CODE REVIEW

File: Yona/Yona/Others/SMSValidPasscodeLogin/LoginSignupValidationMasterView.swift [hunk 1]
```diff
@@ -77,29 +76,26 @@ extension LoginSignupValidationMasterView {
...
                     }
-                } else {
-                    //TODO: Will change this after this build
-                    self.displayAlertMessage("Error", alertDescription: "User not found")
                 }
-            })
-        }
+            } else {
+                //TODO: Will change this after this build
+                self.displayAlertMessage("Error", alertDescription: "User not found")
```
Comment (vasakinnar) [inline Yona/Yona/Others/SMSValidPasscodeLogin/LoginSignupValidationMasterView.swift]: @chandanvarma009 "Error" & "User not found" - use from localization file rather to add here.

Comment (darthpelo) [inline Yona/Yona/Others/SMSValidPasscodeLogin/LoginSignupValidationMasterView.swift]: I am fixing this.

Comment (benSmith1981) [inline Yona/Yona/Others/SMSValidPasscodeLogin/LoginSignupValidationMasterView.swift]: Or from the server if this is part of an API call...

Comment (benSmith1981) [inline Yona/Yona/Others/SMSValidPasscodeLogin/LoginSignupValidationMasterView.swift]: Did you fix it @darthpelo ? Can i merge

Comment (chandanvarma009) [inline Yona/Yona/Others/SMSValidPasscodeLogin/LoginSignupValidationMasterView.swift]: yes its fixed.

**Pull request merged**